.changeset/loud-beds-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-user-settings': major
```

Comment:
Let's make this a `patch` bump. There's no API change and I'd consider the behavior change a fix.